src/rules/no_public_attributes.ts

```diff
@@ -10,7 +10,7 @@ import {Visibility} from "../abap/types";
 
 /** Checks that classes don't contain any public attributes */
 export class NoPublicAttributesConf extends BasicRuleConfig {
-// todo,  public allowReadOnly: boolean = false;
+  public allowReadOnly: boolean = false;
```

Comment:
this not implemented yet, hence the commented code

Reply:
I think I was just trying to figure out if it works and forgot to comment it back.